imageeditlibrary/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 24
+    buildToolsVersion '25.0.0'
```

Comment:
Change the `buildToolsVersion` to 25.0.3 which is mentioned in `travis.yml` to fix the fail checks